docs/API.md

```diff
@@ -312,5 +412,88 @@ Or `.static()` ...
 Or even `.compose()` ...
 
 ```js
-  var obj = abstractStamp.compose(concreteStamp, additionalStamp, utilityStamp).create();
+  const obj = abstractStamp.compose(concreteStamp, additionalStamp, utilityStamp).create();
+```
+
+
+## Chaining methods
```

Comment:
One thing at a time Tim. I'm not changing text at this point of time. Just moving.

There are much more changes to ES6-ify examples than just var->const. If you want to ES6-ify them all I welcome you to supply PR.

This issue name is **Remove api from readme**. It doesn't say "make all examples ES6".

If no other objections present, I'll merge this in few hours.